Bump to v3 API (#59)
nightfallai/jenkins_test: README.md

### Nightfall_Code_Scanner - a code review tool that protects you from committing sensitive information

The `nightfall_code_scanner` is a code review tool that protects you from committing sensitive information to your
version control. It is intended to be used as a part of your CI to simplify the development process, improve your
security, and ensure you never accidentally leak secrets or other sensitive information via an accidental commit.

## Supported Services
* [GitHub Action](https://github.com/nightfallai/nightfall_dlp_action)
* [CircleCI Orb](https://github.com/nightfallai/nightfall_circle_orb)

> ## NightfallDLP Config File

Reviewer: should we add the default here at the top?

Author: done

> The `.nightfalldlp/config.json` file is a centralized config file that defines which detectors to use when scanning

Reviewer: ```suggestion
The `.nightfalldlp/config.json` file contains configuration to define which detectors to use when scanning
```

> content from pull requests. It supports the following configurations:

Reviewer: ```suggestion
content from pull requests. It supports the following options:
```

### Detection Rule UUIDs

> A Detection Rule UUID is a unique identifier for a [Detection Rule](https://docs.nightfall.ai/docs/entities-and-terms-to-know#detection-rules)

Reviewer: can we link to the detection rules page here?

Consider scanning the payload `"my ssn is 678-99-8212"` against a detection rule containing the `CREDIT_CARD_NUMBER` and
`US_SOCIAL_SECURITY_NUMBER` detectors with the logical op `ANY`. In this case, Nightfall would return a finding for the
string `678-99-8212` from the SSN detector. However, scanning this same payload with a logical op of `ALL` will not
return any findings, since the string does not contain any credit card numbers.

### Detector

A detector represents the atomic unit of scanning for sensitive data. The simplest examples of detectors leverage
Nightfall's pre-built detectors for data types such as `CREDIT_CARD_NUMBER` or `API_KEY`. The list of available
detectors is available [here](https://docs.nightfall.ai/docs/detector-glossary). As an example:

```json
  {
    "detector": {
      "detectorType": "NIGHTFALL_DETECTOR",
      "nightfallDetector": "API_KEY",
      "displayName": "apiKeyDetector",
      "minNumFindings": 1,
      "minConfidence": "LIKELY"
    }
  }
```

Two common configurations across all types of detectors are:
- `minNumFindings`: the number of findings that must match the input string in order for Nightfall to return a finding

> - `minConfidence`: the minimum confidence threshold that Nightfall must meet in order to return a finding.

Reviewer: maybe add type information here?
